tests/robot-tests/tests/libs/no_javascript.py

```diff
@@ -0,0 +1,67 @@
+import requests
+from bs4 import BeautifulSoup
+
+def user_gets_parsed_html_from_page(url):
+    response = requests.get(url)
+    return BeautifulSoup(response.text, "html.parser")
+
+
+def user_checks_page_first_accordion_heading_is_not_a_button(parsed_page):
+    first_accordion = parsed_page.select_one("[data-testid='accordionSection']")
+    if not first_accordion:
+        raise AssertionError('No accordion section found')
+
+    heading = first_accordion.select_one('#publications-1-heading')
```

Comment:
Too much duplication, split out another keyword to get the heading, and then re-use that in these heading keywords.